docs/tutorials/load-test-grpc/ghaction.md

```diff
@@ -23,36 +21,55 @@ To use the action to benchmark and validate a gRPC service, use the [load-test-g
 # Run Iter8 experiment
 - uses: iter8-tools/iter8-action@v1
   with:
-    chart: load-test-http
+    chart: load-test-grpc
     valuesFile: myvalues.yaml
 ```
 
-If the experiment has an error or any service-level objectives (SLOs) cannot be validated, the action will fail. The exception is when the option `validateSLOs` is `false`. In this case, only an execution error will result in failure.
+## Experiment configuration
+
+The [basic usage tutorial](basicusage.md) describes various configurable parameters of the `load-test-grpc` experiment. Their default values are documented in the chart's `values.yaml` file and can be viewed as follows.
+
+```shell
+iter8 hub -c load-test-grpc
+cat load-test-grpc/values.yaml
+```
```

Comment:
Wouldn't a reference to the README be a simpler way to do this?

Reply:
README doesn't describe the default values. `values.yaml` is the standard way of doing this in Helm, which we are inheriting.

Reply:
I think you're referring to the chart's README. 

Actually, I think we will want to get rid of the duplicated content in the chart's README, and simply link to iter8.tools and GH Actions documentation from there (so that there is no repetition).

Reply:
Even better, I am thinking we should get rid of the Chart README's altogether :) 

Minimize repetition, and also ensure the flow of dependencies is linear as much as possible ... 

`iter8` -> `hub` -> `iter8-action` <-> `docs`

Reply:
We have some work to do to figure out how to linearize dependencies and avoid repetitious docs... ideally, we also don't have the arrow back from docs to iter8-action.